fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1989,9 +1989,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(state, indexIdToMeta, indexNameToId, keysType, partitionInfo, idToPartition,
```

Comment:
1. selectedIndex should be add
2. do we need to hash all these attributes? I think baseIndexId and SelectedIndexId are enough

Reply:
Why we should use selectedIndex as hash code？equals function contains it

Reply:
if equals function contains selectedIndex, it is safe to remove selected index from hashCode function